unified_targets/docs/Manufacturers.md

```diff
@@ -16,4 +16,4 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |HBRO|Holybro|http://www.holybro.com/index.html|
 |MTKS|Matek Systems|http://www.mateksys.com/|
 |RCTI|RCTimer|http://rctimer.com/|
-|SPRO|Seriously Pro Racing (SP Racing)|http://seriouslypro.com/|
+|SPRO|Seriously Pro Racing (SP Racing)|http://seriouslypro.com/|
```

Comment:
There is some stray change in here. Can you please undo the change to this file?

Reply:
I will fix it. look like I made it worse.